python/src/aac/plugins/validators/exclusive_fields/_exclusive_fields.py

```diff
@@ -28,8 +33,17 @@ def validate_dict(dict_to_validate: dict) -> None:
         present_exclusive_fields = set(validation_args).intersection(set(dict_to_validate.keys()))
 
         if len(present_exclusive_fields) > 1:
-            multiple_exclusive_fields = f"Multiple exclusive fields are defined '{present_exclusive_fields}' in: {dict_to_validate}"
-            findings.add_error_finding(definition_under_test, multiple_exclusive_fields, PLUGIN_NAME, 0, 0, 0, 0)
+            _, second, *_ = validation_args
+            multiple_exclusive_fields = (
+                f"Multiple exclusive fields are defined '{present_exclusive_fields}' in: {dict_to_validate}"
+            )
+            second_exclusive_field_lexeme = definition_under_test.get_lexeme_with_value(second)
+            findings.add_error_finding(
+                definition_under_test,
+                multiple_exclusive_fields,
+                PLUGIN_NAME,
+                *FindingLocation.from_lexeme(PLUGIN_NAME, second_exclusive_field_lexeme).to_tuple(),
+            )
```

Comment:
Future food for thought, we could alter the findings methods to accept a lexeme and do the source location conversion under the hood.